entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityAccessControlHandler.kt

```diff
@@ -127,6 +127,39 @@ class EntityAccessControlHandler(
         )
     }
 
+    @GetMapping("/groups", produces = [MediaType.APPLICATION_JSON_VALUE, JSON_LD_CONTENT_TYPE])
+    suspend fun getGroupsMemberships(
```

Comment:
we hadn't said that it wasn't very necessary as there are not many groups

Reply:
no, we said that yes, there should not have many groups but for consistency with  other "query APIs", we need to have pagination on this API also (even said that after this PR, pagination will be refactored to avoid all the copy / pasted lines).

Reply:
ok i put it back